website/static/js/registrationEditorExtensions.js

```diff
@@ -142,7 +142,19 @@ var Uploader = function(data) {
     $.extend(self, data);
 };
 
+var AuthorImport = function(data, $root) {
+    var self = this;
+
+    self._orig = data;
+    self.contributors = $root.contributors();
+    self.currentQuestion = $root.currentQuestion;
+    self.currentQuestion($root.find('authorship'));
```

Comment:
In this context though the `data` argument refers to the context of the 'authorship' question. I don't think there's necessarily a problem with modifying the root's `currentQuestion`, but it seems unnecessary given the information provided via the existing context.